src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/CacheCollectorPass.php

```diff
@@ -11,51 +11,17 @@
 
 namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
 
-use Symfony\Component\Cache\Adapter\TagAwareAdapterInterface;
-use Symfony\Component\Cache\Adapter\TraceableAdapter;
-use Symfony\Component\Cache\Adapter\TraceableTagAwareAdapter;
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Definition;
-use Symfony\Component\DependencyInjection\Reference;
+@trigger_error(sprintf('%s is deprecated since version 4.2 and will be removed in 5.0. Use Symfony\Component\Cache\DependencyInjection\CacheCollectorPass instead.', CacheCollectorPass::class), E_USER_DEPRECATED);
```

Comment:
All deprecation messages should be updated to what is done elsewhere:
`The "%s" class is deprecated since Symfony 4.2, use "..." instead.`